Makefile

```
@@ -1,256 +1,188 @@
-	podman rm --force --ignore $(STORAGE_CONTAINER_NAME)
-
-.PHONY: storage
-storage: stop-storage start-storage wait-storage
-
-start-storage:
-	podman run -d --rm -ti \
-		--name $(STORAGE_CONTAINER_NAME) \
-		--pod $(POD_NAME) \
-		-e MINIO_ACCESS_KEY=$(STORAGE_ACCESS_KEY) \
-		-e MINIO_SECRET_KEY=$(STORAGE_ACCESS_SECRET) \
-		-e MINIO_DEFAULT_BUCKETS=$(STORAGE_BUCKET):public \
-        $(STORAGE_IMAGE)
-
-wait-storage:
-	$(call wait-for, localhost:9000)
-
-.PHONY: stop-database
-stop-database:
-	podman rm --force --ignore $(DATABASE_CONTAINER_NAME)
-
-.PHONY: database
-database: stop-database start-database wait-database
-
-start-database:
-	podman run -d --rm -ti \
-		--name $(DATABASE_CONTAINER_NAME) \
-		--pod $(POD_NAME) \
-		-e POSTGRES_PASSWORD=$(POSTGRES_PASSWORD) \
-		-e POSTGRES_USER=$(POSTGRES_USER) \
-		-e POSTGRES_DB=$(POSTGRES_DB) \
-		$(POSTGRES_IMAGE)
-
-wait-database:
-	$(call wait-for, localhost:5432)
-
-load-database: set-run-variable-values
-ifneq ("$(wildcard $(DATABASE_RESTORE_FILE))","")
-	podman cp $(DATABASE_RESTORE_FILE) $(DATABASE_CONTAINER_NAME):/mnt/dump_file
-	podman exec $(DATABASE_CONTAINER_NAME) bash -c "pg_restore -v -c -h localhost -U $(POSTGRES_USER) -d $(POSTGRES_DB) /mnt/dump_file || true"
-else
-	@echo "cannot restore because file does not exists '$(DATABASE_RESTORE_FILE)'"
-	@exit 1
-endif
-
-set-run-variable-values:
-	cp --no-clobber contrib/sample.env envvars || true
-	$(eval POD_NAME=run-$(POD_NAME))
-	$(eval DATABASE_CONTAINER_NAME=run-$(DATABASE_CONTAINER_NAME))
-	$(eval ELASTICSEARCH_CONTAINER_NAME=run-$(ELASTICSEARCH_CONTAINER_NAME))
-
-.PHONY: sql
-sql: set-run-variable-values
-	podman run --rm -ti \
-		--pod $(POD_NAME) \
-		$(POSTGRES_IMAGE) psql -h localhost -U $(POSTGRES_USER) $(POSTGRES_DB)
+		--ignorefile .gitignore \
+        	-f scripts/Dockerfile $(PWD)
+
+$(FILES_DIR)/$(IMAGE_NAME)-$(IMAGE_TAG).tar:
+	podman save -o $(FILES_DIR)/$(IMAGE_NAME)-$(IMAGE_TAG).tar $(IMAGE_NAMESPACE)/$(IMAGE_NAME):$(IMAGE_TAG)
+
+.PHONY: carrega-images 
+carrega-images: $(FILES_DIR)/$(IMAGE_NAME)-$(IMAGE_TAG).tar
+	minikube --profile $(CLUSTER_NAME) image load --overwrite=true -v=2 --alsologtostderr $(FILES_DIR)/$(IMAGE_NAME)-$(IMAGE_TAG).tar
+
+uninstall-tika:
+	- $(helm) uninstall tika
+
+install-tika: 
+	$(helm_install) --namespace tika tika tika/tika
+
+.PHONY: tika
+tika:   uninstall-tika install-tika
+
+.PHONY: postgresql
+postgresql:
+	$(helm_install) --namespace postgresql --values $(SCRIPT_DIR)/postgresql-values.yaml --version 12.10.0 postgresql bitnami/postgresql 
+
+.PHONY: uninstall-postgresql
+uninstall-postgresql:
+	- $(helm) uninstall --wait --namespace postgresql postgresql
+
+.PHONY: uninstall-minio
+uninstall-minio:
+	- $(helm) uninstall --wait --namespace minio minio
+
+.PHONY: minio
+minio:  uninstall-minio
+	$(helm_install) --namespace minio --values $(SCRIPT_DIR)/minio-values.yaml --version 12.8.15 minio bitnami/minio 
+
+uninstall-elasticsearch: 
+	- $(helm) uninstall elastic-operator
```

Encontrei um comando que não funciona
```suggestion
uninstall-elasticsearch: 
	- $(helm) uninstall --namespace elastic-system elastic-operator
```